proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -42,6 +42,18 @@ enum OrderBy {
   ORDER_BY_NAME = 3;
 }
 
+enum ElementType {
```

Comment:
Everything we add needs documentation. Among the things that need to be addressed: What is the purpose of this? Why does this not already exist, and why did we need to add a new type?

Reply:
discussed offline for the comments here